### PR TITLE
[Doc] Revise INSERT reference

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — DISMISSED** [review]

File: docs/sql-reference/sql-statements/data-manipulation/insert.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ INSERT { INTO | OVERWRITE } [db_name.]<table_name>
...
+| column_name   | The name of the destination column(s) to load data in. It must be set as columns exist in the destination table. The destination columns you specify are mapped one on one in sequence to the columns of the source table, regardless of what the destination column names are. If no destination column is specified, the default value is all columns in the destination table. If the specified column in the source table does not exist in the destination column, the default value will be written to this column, and the transaction will fail if the specified column does not have a default value. If the column type of the source table is inconsistent with that of the destination table, the system will perform an implicit conversion on the mismatched column. If the conversion fails, a syntax parsing error will be returned. |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/insert.md:31]: grammar: It must be set as columns that exist in xxx

File: docs/sql-reference/sql-statements/data-manipulation/insert.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Query OK, 5 rows affected, 2 warnings (0.05 sec)
 
 ## Usage notes
 
-- As for the current version, when StarRocks executes the INSERT INTO statement, if there is any row of data mismatches the target table format (for example, the string is too long), the INSERT transaction fails by default. You can set the session variable `enable_insert_strict` to `false` so that the system filters out the data that mismatches the target table format and continues to execute the transaction.
+- As for the current version, when StarRocks executes the INSERT INTO statement, if there is any row of data mismatches the destination table format (for example, the string is too long), the INSERT transaction fails by default. You can set the session variable `enable_insert_strict` to `false` so that the system filters out the data that mismatches the destination table format and continues to execute the transaction.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/insert.md:52]: grammar: if any row of data mismatches the destination table format

File: docs/sql-reference/sql-statements/data-manipulation/insert.md [hunk 3]
```diff
@@ -66,9 +66,9 @@ INSERT INTO test (c1, c2) VALUES (1, DEFAULT);
 INSERT INTO test (c1) VALUES (1);
 ```
 
-When no target column is specified, the columns are loaded in sequential order into the target table by default. Therefore, in the above example, the outcomes of the first and second SQL statements are the same.
+When no destination column is specified, the columns are loaded in sequential order into the destination table by default. Therefore, in the above example, the outcomes of the first and second SQL statements are the same.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/insert.md:69]: suggestion: in sequential order --- in sequence

File: docs/sql-reference/sql-statements/data-manipulation/insert.md [hunk 4]
```diff
@@ -66,9 +66,9 @@ INSERT INTO test (c1, c2) VALUES (1, DEFAULT);
...
 
-If there is a target column with no data inserted or data inserted using DEFAULT as the value, the column will use the default value as the loaded data. Therefore, in the above example, the outcomes of the third and fourth statements are the same.
+If there is a destination column with no data inserted or data inserted using DEFAULT as the value, the column will use the default value as the loaded data. Therefore, in the above example, the outcomes of the third and fourth statements are the same.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/insert.md:71]: If a destination column (with or without data inserted) uses DEFAULT as the value

File: docs/sql-reference/sql-statements/data-manipulation/insert.md [hunk 5]
```diff
@@ -66,9 +66,9 @@ INSERT INTO test (c1, c2) VALUES (1, DEFAULT);
...
 
 - Load multiple rows of data into the`test`table at one time.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/insert.md:73]: the`test`table -- missing spaces

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**